app/config/ServiceConfiguration.scala

```diff
@@ -42,26 +42,24 @@ trait ServiceConfiguration {
   def endToEndProcessingThreshold(): Duration
 }
 
-class PlayBasedServiceConfiguration @Inject()(configuration: Configuration, env: Environment) extends ServiceConfiguration {
+class PlayBasedServiceConfiguration @Inject()(configuration: Configuration, runMode: RunMode) extends ServiceConfiguration {
```

Comment:
Do we still need RunMode? can this be done just using the per-environment config instead?

Reply:
Had been attempting to limit the scope of the change, but I suppose now is as good a time as any to finally clean this up.